Packs/PrismaCloudITAdmin/Integrations/Prisma_Cloud_IT_Admin/Prisma_Cloud_IT_Admin.yml

```diff
@@ -0,0 +1,268 @@
+category: IT Services
+commonfields:
+  id: Prisma Cloud IT Admin
+  version: -1
+configuration:
+- defaultvalue: ""
+  display: 'Prisma Cloud url (Eg: https://<domain>.redlock.io/)'
+  name: url
+  required: true
+  type: 0
+- defaultvalue: ""
+  display: Username
+  name: credentials
+  required: true
+  type: 9
+- defaultvalue: ""
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- defaultvalue: ""
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: ""
+  display: CustomerName
+  name: customerName
+  required: true
+  type: 0
+- additionalinfo: An optional custom mapping that takes custom values in the SCIM
+    data into the integration.
+  defaultvalue: ""
+  display: Custom Mapping for Create User
+  name: customMappingCreateUser
+  required: false
+  type: 12
+- additionalinfo: An optional custom mapping that takes custom values in the SCIM
+    data into the integration.
+  defaultvalue: ""
+  display: Custom Mapping for Update User
+  name: customMappingUpdateUser
+  required: false
+  type: 12
+description: 'The Prisma Cloud API consists of a set of API endpoints that allow customers
+  to perform CRUD operation on their user profiles. '
+display: Prisma Cloud IT Admin
+name: Prisma Cloud IT Admin
+script:
+  commands:
+  - arguments:
+    - description: SCIM content in JSON format
+      name: scim
+      required: true
+    description: Get a user
+    name: get-user
+    outputs:
+    - contextPath: GetUser
+      description: Command context path
+      type: Unknown
+    - contextPath: GetUser.active
```

Comment:
for all commands, all context outputs should start with the integration name, e.g `PrismaCloudITAdmin.GetUser.Active`.